spec/controllers/topics_controller_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe TopicsController, type: :controller do
+  include ContentSchemaHelpers
```

Comment:
Do you need this here?

Reply:
I think so, it provides the `example_content_schema` method which we use in these tests: https://github.com/alphagov/govuk-topic-pages-prototype/pull/9/files#diff-4bcc73cef58214df200f492de3d9606fR23

Reply:
Oh! That's a later commit. The line is added in 3333032f955e85909e5a631f4c701c7d0e455dc3. I'd move this statement to the later commit.